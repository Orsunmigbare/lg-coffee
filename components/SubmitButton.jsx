import React from 'react';

const SubmitButton = () => (
  <button className="btn btn-outline-light align-self-end mt-3" type="submit">
    Submit
  </button>
);

export default SubmitButton;
