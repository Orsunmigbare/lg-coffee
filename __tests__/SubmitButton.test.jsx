import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import SubmitButton from '../components/SubmitButton';

it('SubmitButton component renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<SubmitButton />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('SubmitButton component renders unchanged', () => {
  const tree = renderer.create(<SubmitButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
