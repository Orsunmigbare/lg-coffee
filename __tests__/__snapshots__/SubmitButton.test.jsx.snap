// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitButton component renders unchanged 1`] = `
<button
  className="btn btn-outline-light align-self-end mt-3"
  type="submit"
>
  Submit
</button>
`;
